dnf-behave-tests/createrepo_c/update-update-md-path.feature

```diff
@@ -0,0 +1,122 @@
+Feature: Tests createrepo_c --update with --update-md-path
+
+
+Background: Prepare side repository folder
+Given I create directory "/main-repo/"
+  And I copy file "{context.scenario.repos_location}/createrepo_c-ci-packages/x86_64/package-0.2.1-1.fc29.x86_64.rpm" to "/main-repo"
+
+
+Scenario: --update-md-path doesn't work without --update
+ When I execute createrepo_c with args "--update-md-path /i/lead/nowhere ." in "/main-repo"
+ Then stderr matches line by line
+      """
+      \*\* \(createrepo_c:[0-9]*\): WARNING \*\*: .*: Usage of --update-md-path without --update has no effect!
+      """
+  And the exit code is 0
+  And stdout matches line by line
+      """
+      Directory walk started
+      Directory walk done - 1 packages
+      Temporary output repo path: ./.repodata/
+      Preparing sqlite DBs
+      Pool started \(with 5 workers\)
+      Pool finished
+
```

Comment:
... You've got this extra line here... And there's a `strip()` on both `expected` and `found` in the step, so this happens to work. But seems like we're too loose, should we do something about it?

Reply:
I see, I have the extra line in all the stdouts... I agree that the `expected` should be without the strip, that just allows us to be more sloppy.